packages/apps/hr/src/components/templates/BillableSwitchCard/BillableSwitchCard.tsx

```diff
@@ -25,6 +25,8 @@ const BillableSwitchCard = ({
   title = 'Hr_ToInvoice',
   defaultValue = null,
   onChange = () => {},
+  required = true,
```

Comment:
I think it's better to set it to false here and set props to true when we use the component, like we do for other components with required props